tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -5899,6 +5900,61 @@ public StageSize stageSize(List<DataType<?>> dtypes, StageSize.Options... option
     return StopGradient.create(scope, input);
   }
 
+  /**
+   * Return a strided slice from `input`.
+   *   <p>
+   *   The goal of this op is to produce a new tensor with a subset of
+   *   the elements from the `n` dimensional `input` tensor. The subset is chosen using
+   *   a sequence of `m` sparse range specifications encoded into the arguments
+   *   of this function. Note, in some cases
+   *   `m` could be equal to `n`, but this need not be the case. Each
+   *   range specification entry can be one of the following:
+   *   <p>
+   *   - An ellipsis (...) using {@link Index#ellipses()}. Ellipses are used to imply zero or more
+   *     dimensions of full-dimension selection and are produced using
+   *     `ellipsis_mask`. For example, `foo[...]` is the identity slice.
```

Comment:
We don't have this kind of overloading in Java, so does it make sense to describe it here in the Java Ops?

Reply:
Agreed.  I copied most of this from the `stridedSlice` docs and tried to keep it as close as possible.  I talk enough about the Python and Java equivalents that I think it's understood to be `stridedSlice(foo, Index.ellipsis())`.  I'm kind of using it as a generic "slice notation" rather than specifically referencing the Python syntax.